Dockerfile.7.x-cli

```diff
@@ -24,9 +25,11 @@ RUN set -eux \
     &&  touch /etc/msmtprc \
     &&  chmod 640 /etc/msmtprc \
     &&  curl https://packages.sury.org/php/apt.gpg -o /etc/apt/trusted.gpg.d/php-sury.gpg \
-    &&  echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" | tee -a /etc/apt/sources.list.d/php-sury.list \
+    &&  echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php-sury.list \
     &&  apt-get update \
+    &&  cat /etc/apt/preferences.d/php \
     &&  apt-get install -y --no-install-recommends \
+                  php7.${PHP_MINOR_VERSION}-common \
```

Comment:
Not necessary, SAPIs like php7.x-cli always depend on php7.x-common, no need to explictly install it.

Reply:
it's a loosy try to fix pinning bug.